.github/workflows/CI.yml

```diff
@@ -41,7 +41,7 @@ jobs:
           sudo apt-get install -yqq libgeos-dev
       - name: Install Geos (Mac)
         if: matrix.os == 'macos'
-        run: brew install geos
+        run: HOMEBREW_NO_INSTALL_CLEANUP=1 brew install geos
```

Comment:
what's the need for this?

Reply:
We don't need to cleanup since we'll get rid of the instance later on, and that's one step less in the CI :)